frontend/assets/style/_typography.scss

```diff
@@ -76,42 +75,35 @@ $lineHeight: 3, 2.25, 1.6875, 1.3125, 1;
 }
 
 @include until($tablet) {
-  .is-1,
-  h1 {
+  .is-title-1 {
     font-size: $size-2; // 24px
     line-height: 2.125rem; // 34px
   }
 
-  .is-2,
-  h2 {
+  .is-title-2 {
     font-size: 1.25rem; // 20px
     line-height: 1.75rem; // 28px
   }
 }
 
 @include phone {
-  h1 {
+  .is-title-1 {
     font-size: 1.25rem; // 20px
     line-height: 2.125rem; // 34px
   }
 
-  h2 {
+  .is-title-2 {
```

Comment:
Do we not need corresponding `.is-title-3` here and above?

Reply:
only title-1 and title-2 are responsive headings. 3, 4 and 5 are the same everywhere.